packages/core/src/node-utils/manifest.ts

```diff
@@ -62,6 +66,29 @@ export function patchManifest(cwd: string, data: Partial<Manifest>): boolean {
   return true;
 }
 
+export function createOrUpdateManifest(
+  cwd: string,
+  data: Partial<Manifest>
+): void {
+  const maybeManifest: FileContent<Manifest> = retrieveManifest(cwd);
+
+  const updatedManifest = pipe(
```

Comment:
no need for cast pipe + fold, use directly getOrElseW if you return directly the valid structure and do nothing in the Success handler.